DESCRIPTION

```diff
@@ -1,5 +1,5 @@
 Package: r2dii.analysis
-Title: Tools for Climate Scenario Analysis
```

Comment:
Is there a particular reason to change the title? 
r2dii.data is titled: "Datasets to Align Financial Markets with Climate Goals"
r2dii.match is titled: "Tools to Match Financial Portfolios with Climate Data" 

i think for consistency, a title like: "Tools to Calculate Climate Targets for Financial Portfolios"
or "Tools to Align Financial Portfolios with Climate Targets" 
or something like that. I guess I'm being a bit picky